controllers/controllers.js

```diff
@@ -0,0 +1,56 @@
+const User = require("../models/User");
+const Foundation = require("../models/Foundation");
+
+const createUser = async (req, res, next) => {
+  console.log(req.body.role);
+  if (req.body.role === "user") {

```

Comment:
This two conditions are almost the same, you can use one try - catch and add conditions in try 
calling their respective model, something like this: 

`try {
      let newUser;
      if (role === 'user') {
        newUser = await new User(req.body)
      } else {
        newUser = await new Foundation(req.body)
      }
      await newUser.save()
      res.status(201).json(newUser)
} catch (err) {
      console.log("Validation Error:", err.errors);
      res.status(422).json(err.errors);
}`